src/running-a-sora-testnet-node.md

```diff
@@ -0,0 +1,299 @@
+
+## Prerequisites
+
+You will need:
+
+- A machine with Linux, Windows, or macOS
+- A static publicly accessible IP address with open `1337` port 
+- [Docker](https://docs.docker.com/get-docker/) (preferably the latest version). Follow the installation guide for your operating system.
+- At least 128 MB RAM dedicated to a single node container.
+- At least 4GB free space for a single node container.
+
+To check that Docker is installed, run the `docker --version` command in your terminal. You will get an output like this:
+
+```txt
+Docker version 27.5.1, build 9f9e405
+```
+
+Check the container with the `docker run hello-world` command in terminal. If everything works fine, Docker will pull the hello-world image and run it.
+
+::: details Expand to see the `docker run hello-world` output
+
+```
+C:\Users\user>docker run hello-world
+docker: error during connect: Head "http://%2F%2F.%2Fpipe%2FdockerDesktopLinuxEngine/_ping": EOF.
+See 'docker run --help'.
+
+C:\Users\user>docker run hello-world
+Unable to find image 'hello-world:latest' locally
+latest: Pulling from library/hello-world
+e6590344b1a5: Download complete
+Digest: sha256:d715f14f9eca81473d9112df50457893aa4d099adeb4729f679006bf5ea12407
+Status: Downloaded newer image for hello-world:latest
+
+Hello from Docker!
+This message shows that your installation appears to be working correctly.
+
+To generate this message, Docker took the following steps:
+ 1. The Docker client contacted the Docker daemon.
+ 2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
+    (amd64)
+ 3. The Docker daemon created a new container from that image which runs the
+    executable that produces the output you are currently reading.
+ 4. The Docker daemon streamed that output to the Docker client, which sent it
+    to your terminal.
+
+To try something more ambitious, you can run an Ubuntu container with:
+ $ docker run -it ubuntu bash
+
+Share images, automate workflows, and more with a free Docker ID:
+ https://hub.docker.com/
+
+For more examples and ideas, visit:
+ https://docs.docker.com/get-started/
+```
+
+:::
+
+If something went wrong, please visit the [Docker documentation](https://docs.docker.com/). You can also download Docker from here:
+
+- [macOS](https://docs.docker.com/desktop/setup/install/mac-install/)
+- [Windows](https://docs.docker.com/desktop/setup/install/windows-install/)
+- [Linux](https://docs.docker.com/desktop/setup/install/linux/)
+
+## Run a Node
+
+### 1. Download the Configuration File
+
+Download the `docker-compose.volunteer.yml` [configuration file](https://github.com/hyperledger-iroha/iroha/raw/refs/heads/testnet/2.0.0-rc.1/defaults/docker-compose.volunteer.yml) from the Iroha repository.
+
+In your command line navigate to the folder you placed the file into.
```

Comment:
```suggestion
Navigate to the folder where you placed the file in your command line.
```

Reply:
Resolved in commit 88c29302a00fa18434ec2ba84d3c872148245122